### ISSUE TITLE
AHER Accessibility- Home Page:  1.4.12- Text Spacing- Unable to override text spacing

### ISSUE THREAD

**Issue by phudson-he** [archesproject/arches-her#1251]
**Given** I am a user that has difficulty reading text
**When** I use assistive Technology to increase the spacing of the text to meet WCAG requirements
Then The text remains unedited and remains in the default spacing

Users who have difficulty reading text will often uses assistive technology to change the spacing of text to make it easier to read, the current page does not allow for this change, making the page harder to use for these users
When testing this, I usually use the chrome extension "Text Spacing Editor" which can be found [here](https://chromewebstore.google.com/detail/text-spacing-editor/amnelgbfbdlfjeaobejkfmjjnmeddaoj?hl=en&pli=1)

**1.4.12 - Text Spacing**
Users should be able to override author specified text spacing to improve their reading experience. No loss of content of the functionality is lost by changing the spacing. More detail of the specific line spacing can be found [here](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html)

**Comment by phudson-he** [comment]
Findings so far:

- I tried that Chrome plugin and indeed it doesn't work on any Arches page
- I checked that there were no fixed font sizes (px) or letter-spacing values in the css and there wasn't
- In Chrome dev tools, I manually changed the css letter-spacing and word-spacing and it worked
- I added this [bookmarklet](https://codepen.io/stevef/full/YLMqbo) and it worked
- So far I can only presume that the plugin in question simply does not work on all web sites

After a discussions it was decided to park this and wait for Imogen to come back off leave next week and discuss what her thoughts are on this matter.

**Comment by phudson-he** [comment]
Due to the fact that WCAG mentions the bookmarklet way of doing this, and it works, its been decided to close this issue.
